app/queries/servers/preference.test.ts

```diff
@@ -91,6 +132,12 @@ describe('Preference Queries', () => {
                         name: 'test2',
                         value: 'value2',
                     },
+                    {
+                        user_id: 'user1',
+                        category: Preferences.CATEGORIES.DISPLAY_SETTINGS,
+                        name: 'name_format',
+                        value: 'username',
+                    },
```

Comment:
I added an extra pref of different category because if I'm querying for `ADVANCED_SETTINGS`, 2 out of 3 (since the 3rd is not `ADVANCED_SETTINGS` category), will get returned.